src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -39,6 +39,9 @@ public boolean start() throws Exception {
         System.out.println(templates);
 
         Node node = computer.getNode();
+        if (node == null) {
+            throw new Exception("Computer does not correspond to a live node");
```

Comment:
It's a bit unclear to me what exactly means when `node == null` and how I can reproduce that behavior. I remove a node, and then quickly run the build maybe? 
But yes, the error message may be changed. Is it ok if I change this error message on #6? because on that branch I did the actual implementation for the 'exws' step.

Regarding the `Computer` object, how can I configure a Node with 0 executors? because if I try saving a Node config with 0 executors, it fails with a RuntimeException. IMO, I don't think is the plugin's responsibility to check whether the `Computer` object is null or not.